pkg/operator/controller/garden/garden/components.go

```diff
@@ -753,6 +793,131 @@ func getLoadBalancerServiceAnnotations(garden *operatorv1alpha1.Garden) map[stri
 	return nil
 }
 
+func (r *Reconciler) newVirtualSystem() component.DeployWaiter {
+	return virtualgardensystem.New(r.RuntimeClientSet.Client(), r.GardenNamespace, virtualgardensystem.Values{})
+}
+
+func (r *Reconciler) newGardenerAPIServer(ctx context.Context, garden *operatorv1alpha1.Garden, secretsManager secretsmanager.Interface) (gardenerapiserver.Interface, error) {
+	var (
+		err                error
+		apiServerConfig    *operatorv1alpha1.GardenerAPIServerConfig
+		auditWebhookConfig *apiserver.AuditWebhook
+	)
+
+	if apiServer := garden.Spec.VirtualCluster.Gardener.APIServer; apiServer != nil {
+		apiServerConfig = apiServer
+
+		auditWebhookConfig, err = r.computeAPIServerAuditWebhookConfig(ctx, apiServer.AuditWebhook)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I could do this for the `gardener-apiserver`, but `kube-apiserver` also uses this function. Only doing it for GAPI would make it inconsistent, and also doing it for KAPI would mean that we need to pass `operatorv1alpha1.AuditWebhook` here https://github.com/gardener/gardener/blob/282aed3606884247198243cc24eb38a35246e412/pkg/component/shared/kubeapiserver.go#L97 which is quite ugly since the function is also used in the `Shoot` controller where we don't allow configuration of an audit webhook. Hence, I would prefer to keep it as is for now if you agree.